app/services/statistics/base.rb

```diff
@@ -1,13 +1,15 @@
 module Statistics
   class Base
-    attr_reader :title, :count_by
+    attr_reader :title, :count_by, :url
 
     def initialize(options = {})
-      @id       = options[:id]
-      @title    = options[:title]
-      @search   = options[:search]
-      @url      = options[:url]
+      @id = options[:id]
+      @title = options[:title]
+      @search = options[:search]
       @count_by = options[:count_by]
+      @organization_id = options[:organization_id]
+      @location_id = options[:location_id]
```

Comment:
Maybe we should also default here to current taxonomies just to simplify the call for the default case?

Reply:
That does not make sense to me. This is a class that should not know anything about "what is current context". When you generate a statistic, you know for which context you want it.

Also this would effectively mean that if you explicitly set empty org/loc (any context), the class would fallback to current org/loc which seems inconsistent with the usage where in one org you render a chart for another org.

Reply:
Ok, makes sense to still allow explicitly setting `nil`. 
Technically, the statistics service shouldn't be aware of the context either, maybe we should pass that from the controller to `Statistics.charts`?

Reply:
Yup, that sounds reasonable, even though the Statistics service itself as it stands today is not much reusable since it hardcodes the charts for specific page. Fixed.